### PR TITLE
Add note for description field of package.json

### LINKED ISSUE #556: add "decscription" behavior to using package.json page
screen shot 2016-01-13 at 12 53 15 pm

from @snopeks

### PR DESCRIPTION
**Pull request by ilanbiala**
Fixes #556

### CODE REVIEW

**Comment by ilanbiala** [comment]
@ashleygwilliams @PrototypeAlex would be great to get any feedback on this.

File: content/getting-started/using-a-package.json.md [hunk 1]
```diff
@@ -177,3 +180,5 @@ To learn more about semantic versioning, check out our [Getting Started "Semver"
 
 [1]: /getting-started/semantic-versioning
 [2]: https://opensource.org/licenses/ISC
+
+##
```
Comment (ashleygwilliams) [inline content/getting-started/using-a-package.json.md]: is this a typo perhaps?

**Comment by ashleygwilliams** [comment]
this looks awesome. happy to merge once the nit re the `##` at the bottom is dealt with and everything is squashed :)

thanks so much for contributing! :dolphin: :sparkles:

**Comment by ashleygwilliams** [comment]
cc @snopeks does this help with the support questions you get? please review :)

File: content/getting-started/using-a-package.json.md (hunk 1 above)
Comment (ilanbiala) [inline content/getting-started/using-a-package.json.md]: Yeah, I'll remove

**Comment by ilanbiala** [comment]
@ashleygwilliams I fixed the typo at the bottom. I noticed you added a "needs rebase" label. Can you tell me what you want me to do (I assume you mean rebase with master because my changes are behind)?

**Comment by ilanbiala** [comment]
@ashleygwilliams I looked at the branches again, and I'm still unsure of how the rebase should go, so please let me know what to do for the rebase, assuming the PR is good to go other than that.

**Comment by ashleygwilliams** [comment]
hey @ilanbiala i just meant that if you added another commit i'd want you to squash. this looks great! thanks so much! i will send you a coupon for socks -- what email should i send it to? :ghost: :soccer:

**Comment by ilanbiala** [comment]
Thanks for merging it in. You can send it to Ilan.biala@gmail

**Comment by ilanbiala** [comment]
@ashleygwilliams not sure if maybe it went to spam, but I haven't seen any email. Any issues?

**Comment by ilanbiala** [comment]
@ashleygwilliams any issues with the email? I still haven't received anything and I'm not sure what I'm looking for.

**Pull request merged**